### PR TITLE
Release notes 20220725

### PR DESCRIPTION
**Pull request by esproul**

### CODE REVIEW

File: content/circonus/on-premises/changelog/2022.md [hunk 1]
```diff
@@ -5,6 +5,153 @@ weight: 18
...
+> Metric Search v2 and Metric Clusters have been deprecated for some time.  We
+> plan to remove these deprecated functions soon for the SaaS site, and in the
+> subsequent Inside release. This will affect CAQL checks as well as CAQL
+> Datapoints on graphs. The UI will now show users a warning when one of the
+> deprecated functions is used. Circonus offers a more powerful tag-search
+> feature, exposed as `find()` in CAQL.
+
+* Disallow adding `__check_uuid` or `__name` tags in check tags and ignore them
+  if they are already there. (CIRC-8774)
+* Double `latest` values in `/find` calls will now round less and be more
```
Comment (bdunavant) [inline content/circonus/on-premises/changelog/2022.md:47]:
```suggestion
* Floating point `latest` values in `/find` calls will now round less and be more
```

`latest` returns two values, so using "double" there, while correct, is confusing.

File: content/circonus/on-premises/changelog/2022.md [hunk 2]
```diff
@@ -5,6 +5,153 @@ weight: 18
...
+  accurate. (CIRC-8615)
+* Add `min` mechanism arg to `top` reduction and add `bottom` reduction to
+  `/fetch`. (CIRC-8732)
+* Add `method="min"` to `top()` and `bottom()` CAQL functions. (CIRC-8732)
+* Implement `bottom()` CAQL function. (CIRC-8732)
+* More user-errors in CAQL are reported as 4xx HTTP status codes. (CIRC-7874)
+* Fix bug where doubles were being mistakenly cast to integers when returning
+  the last known raw datapoints in `find/tags` calls. (CIRC-8615)
```
Comment (bdunavant) [inline content/circonus/on-premises/changelog/2022.md:55]: This is the same update as the "double latest" one.   Please merge.

Comment (esproul) [reply content/circonus/on-premises/changelog/2022.md]: Done.

**Review by bdunavant — APPROVED** [review]
LGTM

**Pull request merged**